webtau.json

```diff
@@ -0,0 +1,22 @@
+{
+  "description": "WebTau end-to-end testing across layers",
+  "properties": {
+    "v": "1.53"
+  },
+  "link": "https://github.com/testingisdocumenting/webtau",
+  "dependencies": [
+    "org.testingisdocumenting.webtau:webtau:$v"
+  ],
+  "imports": [
+    "org.testingisdocumenting.webtau.WebTauDsl.*"
+  ],
+  "init": [
+    "org.testingisdocumenting.webtau.browser.BrowserConfig.setBrowserSameDriverInThreads(true)"
+  ],
+  "renderers": {
```

Comment:
in a future version of webtau it would make more sense to add a catch all renderer for any class that implements PrettyPrintable